app/src/main/java/it/ilker/repeatcard/ui/directories/DirectoriesAdapter.kt

```diff
@@ -29,8 +29,8 @@ class DirectoriesAdapter(private val clickListener: DirectoriesListener) :
         // Set delete button to invisible if default directory
         holder.directoryDelete.visibility = if (holder.directory.text == DEFAULT_DIRECTORY_NAME) INVISIBLE else VISIBLE
 
-        holder.directoryView.setOnClickListener { clickListener.directoryClicked(directory.id) }
-        holder.directoryDelete.setOnClickListener { clickListener.itemDeleted(directory.id) }
+        holder.directoryView.setOnClickListener { clickListener.directoryClicked(getItem(holder.adapterPosition).id) }
+        holder.directoryDelete.setOnClickListener { clickListener.itemDeleted(getItem(holder.adapterPosition).id) }
```

Comment:
Instead of these changes, maybe we can just modify how we define the variable `directory` as we already use it. So we can avoid duplication:
```
val directory = getItem(holder.adapterPosition)
```